src/routes/routes.js

```diff
@@ -72,4 +74,9 @@ router.post('/api/v1/bookmarks/:slug', [verifyToken, findUser], Bookmark.createB
 router.get('/api/v1/bookmarks', [verifyToken, findUser], Bookmark.getBookmarks);
 router.delete('/api/v1/bookmarks/:slug', [verifyToken, findUser], Bookmark.deleteBookmark);
 
+router.post('/api/v1/articles/:articleSlug/reports', [verifyToken, Validation.reportValidation], Report.createReport);
```

Comment:
The endpoint for reporting an article, I would rather have it to be `/api/v1/reports` instead of `/api/v1/articles/:articleSlug/reports` since `report` is a separate resource in the system besides, it has its own table in the database. So I don't think it's endpoint should have `articles` in its route

Reply:
`/api/v1/articles/:articleSlug/reports` is a better route. Reports will always be associated with a specific article. A consumer of the API is in a better position to understand what the endpoint does just by looking at the route